src/runtime/Types/EventObject.cs

```diff
@@ -27,24 +27,20 @@ public EventObject(EventInfo info)
         /// </summary>
         public static NewReference tp_descr_get(BorrowedReference ds, BorrowedReference ob, BorrowedReference tp)
         {
-            var self = GetManagedObject(ds) as EventObject;
-
-            if (self == null)
+            if (GetManagedObject(ds) is EventObject self)
             {
-                return Exceptions.RaiseTypeError("invalid argument");
-            }
+                if (ob == null)
+                {
+                    return new NewReference(ds);
+                }
 
-            if (ob == null)
-            {
-                return new NewReference(ds);
-            }
+                if (Runtime.PyObject_IsInstance(ob, tp) > 0)
+                {
+                    return new EventBinding(self.name, self.reg, new PyObject(ob)).Alloc();
+                }
 
-            if (Runtime.PyObject_IsInstance(ob, tp) < 1)
-            {
-                return Exceptions.RaiseTypeError("invalid argument");
             }
-
-            return new EventBinding(self.name, self.reg, new PyObject(ob)).Alloc();
+            return Exceptions.RaiseTypeError("invalid argument");
```

Comment:
Same, validate and quick exit.

Reply:
I agree with this in general, but here this would mean that instead of a single `RaiseTypeError` with a particular string, we need two, which does not look like an advantage to me.

Reply:
I think the old version is easier to read. Nesting makes it harder.

Reply:
Is this your only gripe with this PR? I find the new version easier to read because it puts emphasis on the happy path.

Reply:
Yes. I also think that since we are in disagreement, a change should not be made without some kind of necessity.

Reply:
Meh. I'll revert these particular changes, the rest is important (introducing `readonly` wherever possible etc.).